docs/Queries/About Queries.md

```diff
@@ -23,6 +23,24 @@ In Live Preview and Reading modes, this will list *all* tasks from your vault, r
 This is probably not what you want.
 Therefore, Tasks allows you to set query options to filter the tasks that you want to show.
 
+For instance, you can show only the tasks (from anywhere in the vault) that are due today:
+
+    ## Due today
+    ```tasks
+    due today
+    not done
+    ```
+
+You can create as many task queries as you like, and you can also wrap them into callouts to style them a bit differently:
```

Comment:
Super - thank you.

Do you think it might be useful to link to the callout docs?
https://help.obsidian.md/Editing+and+formatting/Callouts

Reply:
Good idea, did that now.